src/lingrex/trimming.py

```diff
@@ -0,0 +1,201 @@
+"""
+Trimming functionalities in lingrex.
+"""
+import random
+import collections
+from lingpy.sequence.sound_classes import token2class
+
+
+def revert(alms):
+    return [[row[i] for row in alms] for i in range(len(alms[0]))]        
+
+
+def get_skeleton(alms, gap="-"):
+    return [token2class(([c for c in col if c != gap] or ["+"])[0], "cv") for col in revert(alms)]
+
+
+def apply_trim(alms, idxs):
+    """
+    Basic trimming function, based on a selection of indices that are trimmed.
+    """
+    return [[row[i] for i in range(len(row)) if i not in idxs] for row in alms]
+
+
+def subsequence_of(source, target):
+    """
+    Check if source is a subsequence of target.
+    """
+    q_1, q_2 = list(target), list(source)
+    while q_1:
+        s = q_1.pop(0)
+        if q_2 and q_2[0] == s:
+            q_2.pop(0)
+        elif q_2:
+            pass
+        else:
+            break
+    if not q_2:
+        return True
+    return False
+
+
+def consecutive_gaps(seq, gap="-"):
+    """
+    Return consecutive gaps in line.
+    """
+    start, end = [0], [len(seq)]
+    for i in range(len(seq)):
+        if seq[i] == gap:
+            gapped = True
+        else:
+            gapped = False
+        if gapped:
+            start += [i + 1]
+        if not gapped:
+            break
+    for i in range(len(seq) - 1, 0, -1):
+        if seq[i] == "-":
```

Comment:
Just realized that this is probably a bug. Should be `if seq[i] == gap:` rather than comparing to the string literal here.